lib/scsi_defs.h

```diff
@@ -80,5 +85,23 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  //SCSI Reference Manual Table 202 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  typedef struct TestUnitReadyCommand {
+    OpCode opCode = OpCode::kTestUnitReady;
+    uint32_t reserved;
+    ControlByte controlByte;
+    uint16_t padding;   // Necessary for bit_cast
+
+    TestUnitReadyCommand() {}
+
+    TestUnitReadyCommand(std::bitset<48> cmd) {
+      if (OpCode::kTestUnitReady != (OpCode) (cmd.to_ulong() & 8)) {
+        throw std::runtime_error("Invalid OpCode for Test Unit Ready Command");
```

Comment:
I'm wondering if it would be better to use a custom exception class(es) or not

Reply:
Yeah we could make an InvalidOpCodeException (or a different name)